entries/n/note.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Note": {
+    "domain": "note.com",
+    "additional-domains": [
+      "note.jp",
+      "note.mu"
+    ],
+    "contact": {
+      "facebook": "note.poc",
+      "twitter": "note_PR",
+      "email": "info@note.jp"
+    },
+    "keywords": [
+      "social"
+    ]
```

Looks like it's only in Japanese? A region tag might be good.
```suggestion
    ],
    "regions": [
      "jp"
    ]
```

---

Note can be signed up from outside Japan. Language and region should not be confused.

---

I doubt it will be of much use on the Brazilian or German versions of 2fa.directory.

---

Or maybe we should create a separate page for each language, separate from the region-specific pages we already have.